server/app/views/questiontypes/DropdownQuestionFragment.html

```diff
@@ -0,0 +1,27 @@
+<div th:fragment="dropdown(question)"
+     th:with="singleSelectQuestion=${question.createSingleSelectQuestion()},
+              questionId=${#strings.randomAlphanumeric(8)},
+              inputId=${#strings.randomAlphanumeric(8)}"
+     th:id="${questionId}">
+    <!-- Title and Help Text -->
+    <div th:replace="~{questiontypes/QuestionBaseFragment :: titleAndHelpTextSingleInput(${question}, ${questionId}, ${inputId})}"></div>
```

Comment:
question: Should this be a label?

Reply:
It doesn't really matter. we're using th:replace, which removes the <div> I'm adding and replaces it with the <label> rendered by that fragment. I've been using <div> in these cases but not for any particular reason.